packages/zarathustra/skills/derolas_automator_abci_app/behaviours.py

```diff
@@ -293,7 +356,12 @@ def act(self) -> None:
         """Perfom the act."""
 
         try:
-            if self.claimable > 0:
+            game_state = self.game_state
+            self.context.logger.info(f"{self.name}: game state: {game_state}")
+            if not game_state.blocks_remaining:
```

Comment:
Nicely lets us hop out. 
I guess we should have the game state in the strategy save all the calls.